resources/guis.yml

```diff
@@ -274,37 +274,39 @@ guis:
   price: "$99/user, $75 annual business sub, free eval"
   license: Proprietary
   order: 30
+- name: LazyGit
+  url: https://github.com/jesseduffield/lazygit
+  image_tag: guis/lazygit@2x.png
+  platforms:
+  - Windows
+  - Mac
+  - Linux
+  price: Free
+  license: MIT
+  order: 31
```

Comment:
there seems to be a problem with the image: https://git-scm-pr-1496.herokuapp.com/downloads/guis

Reply:
we are missing the normal version besides the 2x. See this PR as an example https://github.com/git/git-scm.com/pull/1485/files